app/models/KafkaMessageReader.scala

```diff
@@ -0,0 +1,107 @@
+package models
+
+import java.io.{ByteArrayInputStream, InputStreamReader}
+import java.nio.ByteBuffer
+import java.util.zip.{GZIPInputStream, ZipException}
+
+import com.google.common.io.CharStreams
+import kafka.api.{FetchRequestBuilder, PartitionMetadata, TopicMetadataRequest}
+import kafka.consumer.SimpleConsumer
+import play.api.Play.current
+import play.api.{Logger, Play}
+
+/**
+  * Contains code to read messages from kafka.
+  */
+object KafkaMessageReader {
+
+  private val Kafkas = Play.configuration.getString("capillary.kafkas").getOrElse("127.0.0.1:9092")
```

Comment:
Suggestion:  I would make this a class and pass these things in rather than having them be constants, it makes it easier to test and a bit more re-usable.  If you wanted to you could make an apply method in the companion object that takes a `Config` option as its argument.